src/test/java/org/broadinstitute/hellbender/utils/BaseUtilsUnitTest.java

```diff
@@ -27,34 +29,49 @@ public void testConvertIUPACtoN() {
         checkBytesAreEqual(BaseUtils.convertIUPACtoN(new byte[]{'M', 'M', 'M'}, false, false), new byte[]{'N', 'N', 'N'});
     }
 
+    @DataProvider
+    Iterator<Object[]> allByteValues() {
+        final List<Object[]> bytes = new ArrayList<>(256);
+        for (int i = 0; i < 256; i++) {
+            final byte oneByte = (byte) i;
+            bytes.add(new Object[]{oneByte});
+        }
+        return bytes.iterator();
+    }
+
+    @Test(dataProvider = "allByteValues")
+    public void testCanConvertAllBytes(final byte b) {
+        BaseUtils.convertIUPACtoN(new byte[]{b, b, b}, false, false);
```

Comment:
Can you also add a call to `BaseUtils.simpleBaseToBaseIndex()` in this test method, to cover the other change you made?

Reply:
will do.